applications/tari_merge_mining_proxy/src/proxy.rs

```diff
@@ -615,13 +619,29 @@ impl InnerService {
         }
 
         trace!(target: LOG_TARGET, "Sending request {:?}", request.body(),);
-        let resp = builder
+
+        let json_response;
+        if submit_block && !self.config.proxy_submit_to_origin {
```

Comment:
I think we can probably print a once-off status message to the proxy console to advocate if blocks will not be submitted to `monerod`.

Reply:
Good idea.